changelog/fragments/1681919580-increase-fleet-long-poll-timeout.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: increase-fleet-long-poll-timeout
```

Comment:
You want this to be a human readable description, the changelog tool doesn't make this clear.

I would state what the timeout has increased to, why, and what effects it might have. In particular this changelog entry should call out that people using non-Elastic proxies may have to adjust their connection timeouts.